Dockerfile

```diff
@@ -57,9 +59,11 @@ RUN python3 -m pip install -e git+https://github.com/kytos-ng/of_core@${branch_o
 # pymongo and requests resolve to the same version on kytos and NApps
 RUN python3 -m pip install pytest-timeout==2.2.0 \
  && python3 -m pip install pytest==8.1.1 \
+ && python3 -m pip install pytest-asyncio==1.1.0 \
  && python3 -m pip install pytest-rerunfailures==13.0 \
  && python3 -m pip install mock==5.1.0 \
  && python3 -m pip install pymongo \
+ && python3 -m pip install aiokafka==0.12.0 \
```

Comment:
`aiokafka` dependency is a `requirements/run.txt` dependency from `kafka_events` that you're introducing on https://github.com/kytos-ng/kafka_events/pull/13 which will be already installed with the prior `pip install -e ...` command above, so you should remove it from here too.